share/functions/help.fish

```diff
@@ -224,10 +230,13 @@ function help --description 'Show help for the fish shell'
         end
     end
 
-    # cmd.exe needs more coaxing.
+    # cmd.exe and powershell needs more coaxing.
     if string match -qr 'cmd\.exe$' -- $fish_browser[1]
         # The space before the /c is to prevent msys2 from expanding it to a path
         $fish_browser " /c" start $page_url
+    else if string match -qr 'powershell\.exe$' -- $fish_browser[1]
```

Comment:
Just adjust the regex above - `if string match -qr 'powershell\.exe$|cmd\.exe$' -- $fish_browser[1]`

Reply:
adjusted regex with `if string match -qr 'powershell\.exe$|cmd\.exe$' -- $fish_browser[1]`